### DISCUSSION TITLE
角色养成计划存在BUG

### DISCUSSION THREAD

**Discussion by itarufan** [LmeSzinc/StarRailCopilot#513] [General]
![QQ图片20240613203625](https://github.com/LmeSzinc/StarRailCopilot/assets/33002055/40e63006-f7ec-45fe-974d-7d52f56d707e)
凭感觉，养成计划判断是否完成100%，是在执行过1次以后才更新数据，会导致接近目标100%完成情况下溢出
比如我这里角色养成材料，在60/65开始执行任务，因为体力足够，脚本直接刷了4次材料本才把体力消耗完，过程中并没有判断是不是满足65/65了，导致最终结果是刷到了80/65材料，有3次是浪费的

**Comment by LmeSzinc** [comment]
有问题发log
